config/hsc/isr.py

```diff
@@ -13,6 +14,9 @@
 config.doDark = True # Required especially around CCD 33
 config.doFringe = True
 config.fringe.filters = ['y', 'N921', 'N926', 'N973', 'N1010']
+
+# Do not use NO_DATA pixels for fringe subtraction.
+config.fringe.stats.badMaskPlanes = ['SAT', 'NO_DATA']
```

Comment:
This looks like an unrelated functional change that should be in a separate commit and justified.

Reply:
This was pulled from `config/hsc/processCcd.py`, which listed these values for `config.isr.fringe.stats.badMaskPlanes`.  Rereading the version on master, I'm now not convinced that the `+= ["SUSPECT"]` line isn't the change in functionality.  I will try to find a test.